discovery-provider/src/tasks/generate_trending.py

```diff
@@ -122,11 +138,17 @@ def generate_trending(db, time, genre, limit, offset):
                 {user_id: follower_count for (user_id, follower_count) in follower_counts}
 
         save_counts = get_save_counts(session, False, True, not_deleted_track_ids, None)
+        save_counts_with_time = get_save_counts_with_time(session, False, True, not_deleted_track_ids, None, time)
         track_save_counts = {
             save_item_id: save_count
             for (save_item_id, save_count, save_type) in save_counts
             if save_type == SaveType.track
         }
+        track_save_counts_with_time = {
```

Comment:
same here, let's make the structure & order the same as reposts

save_counts
track_save_counts
track_save_counts_with_time